src/components/common/Picker/styles.js

```diff
@@ -7,39 +8,45 @@ const baseLabel = {
 };
 
 const basePicker = {
-  width: 230,
-  height: 40,
+  width: '67%',
+  height: 'auto',
+  marginLeft: 'auto',
+  marginRight: 'auto',
+  paddingTop: 12,
```

Comment:
Instead of defining all 4 paddings, can't you use the `padding` shorthand? 

Something like: `padding: 12 5`

The same comment goes for the margin. 
Just googled it and found this: https://stackoverflow.com/questions/48955215/margin-or-padding-shorthand-in-react-native, make your search